zio/zjsonio/ast.go

```diff
@@ -0,0 +1,20 @@
+package zjsonio
+
+import (
+	"github.com/brimdata/zed/compiler/ast"
+	"github.com/brimdata/zed/pkg/unpack"
+)
+
+var unpacker = make(unpack.Reflector)
```

Comment:
```suggestion
var unpacker unpack.Reflector
```

Reply:
This was a bad suggestion because `unpack.Reflector` is a map.

New suggestion: Use `unpack.New`!

Reply:
Newer suggestion: Go back to your original code, which I've done in 4d3ff279f46b9b72a64312cb800ef19a694d07b2.